ES/ESmy020.xml

```diff
@@ -31,71 +31,394 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
                   <idno>MY-020</idno>
                </msIdentifier>
                <msContents>
-                  <summary>I) Maṣḥafa gǝnzat “Book of the Funeral Ritual” (fols. 2ra l.1–53rb
-                     l.10)&#xD; - Homily of St Jacob of Serug on funeral of priests and deacons
-                     (fols. 11rb l.3–15rb l.6)&#xD; - Homily of ʾAbbā Salāmā, Metropolitan of
-                     Ethiopia (fols. 32rb l.10–35rb l.10)&#xD; II) Maṣḥafa krǝstǝnnā “Book of the
-                     Christening”, called here Maṣḥafa ṭǝmqat (fols. 54ra l.1–71rb l.8). For the
-                     title of the book s. e.g., EMML 3107 (EMML VIII, 94)&#xD; &#xD; Additiones
-                     1-2.</summary>
+                  <summary>
+                  </summary>
+
+
                   <msItem xml:id="ms_i1">
+                    <locus from="1r" to="74rb"/>
+                    <title type="complete" ref="LIT1931Mashaf"/>
 
-                     <incipit>
-                        <foreign xml:lang="en"/>
-                     </incipit>
-                     <explicit>
-                        <foreign xml:lang="en"/>
-                     </explicit>
-                     <note/>
+                    <msItem xml:id="ms_il.1">
+                    <locus from="2ra" to="2vb"/>
+                    <title type="incomplete">Mǝlṭān- and ʾaryām-antiphons for priests and deacons</title>
+                    <incipit>
+                    <q xml:lang="gez">
+                      </q></incipit>
+                      <explicit>
+                      <q xml:lang="gez">ፍቁራኒሁ፡ ለብ፡ ኄራን፡ መተንበላ
+                      </q></explicit>
+                    <note>The text is the end of <ref corresp="#ms_il.6"/>; the leaf is placed in the reversed position,
+                    to be read from <foreign xml:lang="gez">ን፡ ኅቡረ፡ ይባርክዎ፡ ለአብ፨</foreign>, <locus target="2va"/>)</note>
                   </msItem>
-                  <!--The data in the items below refers to other items. please remove this items and move the information to the appropriate place.-->
-                  <msItem xml:id="ms_i2">
-                     <note>Text I proper begins on <locus target="#7r">7r</locus> with the first
-                        station.</note>
+
+                    <msItem xml:id="ms_il.2">
+                    <locus from="3ra" to="3rb"/>
+                    <title type="incomplete">Absolution and prayers</title>
+                    <incipit>
+                    <q xml:lang="gez">ሃል፡ ሕዝቦ፡ እስመ፡ ኄር፡ ውእቱ፡ ወመፍቃሬ፡ ሰብእ፡
```

Comment:
no q inside of incipit, as above, should be 

`<incipit xml:lang="gez"><locus target="#3ra"/>ሃል፡ ሕዝቦ፡ እስመ፡ ኄር፡ ውእቱ፡ ወመፍቃሬ፡ ሰብእ፡</incipit>`

Reply:
Done. No idea why i switched to <q/>